app/subscriber.py

```diff
@@ -55,7 +57,7 @@ def receipt_to_case(message: Message):
         log.error('Pub/Sub Message has invalid RFC 3339 timeCreated datetime string')
         return
 
-    log = log.bind(case_id=case_id, created=time_obj_created, tx_id=tx_id)
+    log = log.bind(questionnaire_id=questionnaire_id, created=time_obj_created, tx_id=tx_id, case_id=case_id)
```

Comment:
This line will error if there is no case id on the message

Reply:
dont think it will error as case_id will always be zero if it doesn't exists in the GCP pubsub message - although maybe this is the best approach 🤔

Reply:
Aha, I didn't see that 0 default value, my mistake.  I still think it's worth having the integration tests send in a message with no `case_id` key though.

Reply:
`None` might be a more appropriate default for missing case ID's?

Reply:
ah yes good point, will change to use None, will also change to test too 👍

Reply:
Can probably now remove the lines that follow the new `.get()`: 47-48.

Reply:
fixed